### PR TITLE
Remove: Files-Community.Files version 0.13.0.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
﻿- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/24067)

### CODE REVIEW

File: manifests/f/Files-Community/Files/1.5.0.0/Files-Community.Files.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msix
+  Scope: machine
+  InstallerUrl: https://github.com/files-community/Files/releases/download/v1.5/Files.Package_1.5.0.0_x86_x64.msixbundle
+  InstallerSha256: B552634F7F409F3A5407EE14B2F6CC05C8A37E166961CA146BDF0BFC8E6D2156
+  SignatureSha256: 6d77f3851d1d6a812691661b7c345fe37b748d767da44df6c3c777f791898515
+  FamilyName: Files_1y0xx7n9077q4
```
Comment (ItzLevvie) [inline manifests/f/Files-Community/Files/1.5.0.0/Files-Community.Files.installer.yaml:19]:
```
  FamilyName: Files_1y0xx7n9077q4
```
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
Also, I don't know if this is me (and not sure how the pipelines passed if it isn't), but I'm getting a dependency error:
image

**Comment by vedantmgoyal2009** [comment]
> Also, I don't know if this is me (and not sure how the pipelines passed if it isn't), but I'm getting a dependency error:
> image

If it is really a dependency error, then the whole package should be purged from the repo?

**Comment by ItzLevvie** [comment]
> If it is really a dependency error, then the whole package should be purged from the repo?

Last time I checked, it was only version 1.0.0.0 and above that had the dependency issue, that means that this version has it as well.

The one in the repo which is 0.9.1.0 and older doesn't have it.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/24067#issuecomment-894528337

**Comment by vedantmgoyal2009** [comment]
If it is Azure-Pipeline-Passed, then I don't think there should be any issues.

**Comment by ItzLevvie** [comment]
We might have to wait for @denelon because I tried this PR on the following Windows branches: rs_prerelease, co_release, vb_release & main but all of them are having this dependency issue.

**Comment by OfficialEsco** [comment]
I didn't want to review this because i knew you removed it, however the Dependency is weird, i believe the Dependency is pretty much installed on any OS, however not on a very fresh install/VM/Sandbox

If you run the installer you see nothing but a normal dialog, if you then run winget install it works fine. I believe this could be in the repo as MOST end users will see no issues

My PR which failed https://github.com/microsoft/winget-pkgs/pull/18836
The issue over at Files https://github.com/files-community/Files/issues/5176

**Comment by ItzLevvie** [comment]
>I didn't want to review this because i knew you removed it, however the Dependency is weird, i believe the Dependency is pretty much installed on any OS, however not on a very fresh install/VM/Sandbox

As far as I'm aware, MUX 2.5 isn't installed on any version of Windows unless you downloaded a third-party Microsoft Store application that uses 2.5, so even if you never did a fresh install of Windows and you never downloaded any third-party Microsoft Store applications that uses 2.5, you won't be able to install Files. This was the case for one of my PCs which was running a preview build of Windows in my storage area which I don't use as much anymore.

My main PC which runs a live build of the OS has 2.5, but that's mainly because I use some of the applications from the Microsoft Store developed by [UWP Community](https://uwpcommunity.com/) which utilizes the latest dependencies and libraries, and developers from [UWP Community](https://uwpcommunity.com/) are quick to update their dependencies and libraries whenever Microsoft releases a new update.

Also, I think none of Windows' first-party applications use 2.5. Microsoft jumped from 2.3 and 2.4 to 2.6 for some of their first-party applications, so 2.5 was skipped and isn't installed by default on Windows 10 and 11, and clicking on the "Get updates" button on the Microsoft Store doesn't install any dependencies even if all of the applications were updated. Running `Get-AppxPackage -PackageTypeFilter Framework | Sort-Object Name | Select-Object Name, Publisher, Version, Architecture` still shows that 2.5 wasn't installed from any of the application updates.

I could be wrong because I mainly use Insider Preview builds of the OS which always uses the pre-release versions of everything so I'm ahead of most people nowadays.

>If you run the installer you see nothing but a normal dialog, if you then run winget install it works fine. I believe this could be in the repo as MOST end users will see no issues

This is because the built-in App Installer automatically installs the dependency for you—at least when you manually install the `.msixbundle` (and `.appxbundle`) file(s) using the built-in App Installer on an OS that doesn't have 2.5 installed—but I believe WinGet uses the `Add-AppxPackage` functionality here, which actually requires 2.5 to be downloaded and installed first before Files can be installed on your PC. Perhaps there's an open issue over at winget-cli about this?

Which technically means that if this package was added, no one would be able to do `winget install Files-Community.Files --version 1.5.0.0` because WinGet can't automatically install the dependency, except only if you manually installed it from the repository's releases page using the built-in App Installer or if you downloaded a third-party Microsoft Store application that uses 2.5.

So most users only have one option if they want to install Files—and for other applications—there are two options:
(1) Download and install the application manually from the repository's releases page, or by direct URL if the package was merged, so you can do `winget show Files-Community.Files --version 1.5.0.0 | find "  Download Url: "` to find the application URL and paste it on your browser's address bar.
~(2) Go to `%TEMP%\WinGet` and run the `.appxbundle` or `.msixbundle`~ - unfortunately, this option isn't available for `.appx` and `.msix` applications. It doesn't seem to store the downloaded file there. Most people probably wouldn't know where WinGet downloaded the file to—it's downloaded to `%TEMP%\WinGet`—but they won't know unless they checked the documentation first.

>My PR which failed #18836

I can see the issue on your PR because you used the same URL for x64 on x86. Files cannot run on a 32-bit OS because the developer hasn't compiled the 1.5.0.0 version for the x86 architecture.

`x86_x64` in the URL is `x64` and not actually meant for both architectures.

So, if @denelon sees no issue with this PR then I can approve, but the only thing I can say here is that it won't install until you either have the dependency downloaded or download a third-party Microsoft Store application that uses 2.5.

**Comment by denelon** [comment]
I've asked the team to look into this one. We're getting close to the next release, and I'd like to know if that will resolve this or not. If not, we'll have to wait for the dependency feature to get finished.

**Comment by vedantmgoyal2009** [comment]
Waiting for the next release.

**Comment by vedantmgoyal2009** [comment]
> My main PC which runs a live build of the OS has 2.5, but that's mainly because I use some of the applications from the Microsoft Store developed by [UWP Community](https://uwpcommunity.com/projects) which utilizes the latest dependencies and libraries, and developers from [UWP Community](https://uwpcommunity.com/projects) are quick to update their dependencies and libraries whenever Microsoft releases a new update.

How do you run a live build, please tell 🙏

**Comment by ItzLevvie** [comment]
> How do you run a live build, please tell 🙏

Live = prod(uction).

But for the daily builds of Windows 10 & 11 that I run on some of my PCs, I'll have to point you to https://careers.microsoft.com 😀

**Comment by vedantmgoyal2009** [comment]
> > How do you run a live build, please tell 🙏
> 
> Live = prod(uction).
> 
> But for the daily builds of Windows 10 & 11 that I run on some of my PCs, I'll have to point you to https://careers.microsoft.com 😀

So you mean for daily builds, I will have to be a Microsoft employee but Microsoft will not hire a 14-year-old child 😥. As you have access to daily builds, did you receive Android Apps support in Windows 11?

**Comment by arjun-msft** [comment]
@ItzLevvie are you still seeing issues installing this one?  winget installed this one no problem when I tried and in our validation.

**Comment by ItzLevvie** [comment]
Yes, @KevinLaMS ran into the same issue as me regarding MUX 2.5. I don't know what dependencies the pipelines have installed.

from https://github.com/microsoft/winget-pkgs/pull/5508#issuecomment-745062260:
image

**Comment by yaichenbaum** [comment]
Files 2.0 uses WinUI 2.6 and will be rolling out on October 4th so perhaps it makes sense to wait until then.

**Comment by vedantmgoyal2009** [comment]
Any decisions on this? https://github.com/microsoft/winget-pkgs/pull/24067#issuecomment-915614903

**Comment by yaichenbaum** [comment]
> Can it be merged now? [#24067 (comment)](https://github.com/microsoft/winget-pkgs/pull/24067#issuecomment-915614903)

2.0 will be released in around 2 weeks.

**Comment by vedantmgoyal2009** [comment]
I will update this PR only with manifests of version 2.0, till then I'm marking it as a draft.

**Comment by vedantmgoyal2009** [comment]
Finally, Files 2.0 releases.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
This issue is caused by a dependency not available in the validation pipelines.

from the latest version of WinGet:
image

from an older version of WinGet:
image

**Comment by yaichenbaum** [comment]
Perhaps we can stop using the custom package now that Winget can install Files directly from the store.

**Comment by yaichenbaum** [comment]
> You mean to purge the package from this repo?

Yeah

**Comment by vedantmgoyal2009** [comment]
I will open PRs in a few minutes. By the way, I have also started a discussion #29985 for this since there are many other packages which are shipped through both MS Store and winget.

**Review by ItzLevvie — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**